phoenix-ios/phoenix-ios/views/widgets/AppStatusPopover.swift

```diff
@@ -0,0 +1,620 @@
+import SwiftUI
+import PhoenixShared
+import os.log
+import CircularCheckmarkProgress
+
+#if DEBUG && true
+fileprivate var log = Logger(
+	subsystem: Bundle.main.bundleIdentifier!,
+	category: "AppStatusPopover"
+)
+#else
+fileprivate var log = Logger(OSLog.disabled)
+#endif
+
+
+struct AppStatusPopover: View {
```

Comment:
I wonder if this popover could use instead the swift sheet element (the UI used to display payments). Maybe it's a change is beyond the scope of this PR. But it would provide a more consistent UI, and allow us to add more stuff in this window.

Reply:
If we add more stuff to this window, it might be worth it. I believe the industry prefers to use sheets for **interactive** views (e.g. make a choice to continue), and popover windows for small **informational** views.

However, if the informational view is large (like our payment view), then a full sheet or view is used.

And another exception is if the interactive view is small, the preference is to use short sheets. For example, we recently added a new long-press option to the copy button(s), which display a short sheet:

<img width="250" alt="Screen Shot 2021-08-19 at 10 11 39" src="https://user-images.githubusercontent.com/304604/130094750-37a65ec7-4b66-4d05-90c0-18d8144e4634.png">